examples/usdTieredSTO.ts

```diff
@@ -94,15 +88,34 @@ window.addEventListener('load', async () => {
     value: setupCost,
   });
 
-  // Call to add module
+  // Get General TM Address to whitelist transfers
```

Comment:
Should be changed to `// Get General Transfer Manager to whitelist an address to buy tokens`